packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -870,7 +870,8 @@ File findBundleFile(GradleProject project, BuildInfo buildInfo) {
   return null;
 }
 
-Map<String, String> get _gradleEnv {
+/// The environment variables needed to run Gradle.
+Map<String, String> get gradleEnv {
```

Comment:
`@visibleForTesting` ?

Reply:
The getter is also used from `cache.dart`